tests/networking/nethelper/nethelper.go

```diff
@@ -370,7 +370,11 @@ func getInterfacesList(runningPod corev1.Pod) ([]string, error) {
 	}
 
 	for _, nodeInterface := range linuxInterfaces {
-		if nodeInterface.Master == "" && nodeInterface.Operstate == "UP" {
+		if nodeInterface.Master == "" &&
+			!strings.Contains(nodeInterface.IfName, "ovn") &&
```

Comment:
This fixes the issue where multus if could be created on a non external if, so if shutdown traffic still up. But now I guess even not fixing this since we turn the port off in the pod it would still work

Reply:
yes, right